### PR TITLE
Transparency about accepted grants

### PR DESCRIPTION
**Pull request by Noc2**
Update the grants accepted list to me more transparent.

### CODE REVIEW

File: grants/accepted_grant_applications.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-# Accepted Grant Applications <!-- omit in toc -->
+# Accepted Grant Applications [WIP!] <!-- omit in toc -->
```
Comment (alxs) [inline grants/accepted_grant_applications.md:1]: Do you want to leave this here on purpose? It's more of a living document now and not a WIP

File: grants/accepted_grant_applications.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
 
-This page gives an overview of accepted grants and a link to their GitHubs. Keep in mind that not all of the accepted grants have already delivered software.
+> **:loudspeaker:** This page provides an overview of accepted grant applications, their progress and a link to their GitHub repositories. In cases where the link points to an organization, you should be aware that the grant application itself **is often an independent project unrelated to other work done by the teams**. Besides, **there is a clear difference between an application being accepted and the successful delivery of the respective project**, and only teams that have successfully delivered a milestone are allowed to make public announcements on the matter or to use our [badge](https://github.com/w3f/General-Grants-Program/blob/master/grants/grant-badge-guidelines.md). The badge can also never be used as a general endorsement for a team. Violations to this policy can be reported [here](mailto:grants@web3.foundation).
```
Comment (alxs) [inline grants/accepted_grant_applications.md:3]:
```suggestion
> **:loudspeaker:** This page provides an overview of accepted grant applications, their progress and a link to their GitHub repositories. In cases where the link points to an organization, you should be aware that the grant application itself **is often an independent project unrelated to other work done by the teams**.
```

File: grants/accepted_grant_applications.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
...
+
+> Furthermore, the page lists terminations which happened due to a breach of the terms of the grants programs. Additionally, teams might have decided to stop working on the grant (though not necessarily on the project itself) for various reasons, which are not reflected on this sheet.
```
Comment (alxs) [inline grants/accepted_grant_applications.md:5]: The thought process behind this was that it's not only the reasons that are not reflected on the sheet, but even whether teams have stopped working on a grant or not.

```suggestion
> Furthermore, the page lists terminations which happened due to a breach of the terms of the grants programs. Additionally, teams might have decided to stop working on the grant—though not necessarily on the project itself—for various reasons, which is not reflected on this sheet.

> Besides, **there is a clear difference between an application being accepted and the successful delivery of the respective project**, and only teams that have successfully delivered a milestone are allowed to make public announcements on the matter or to use our [badge](https://github.com/w3f/General-Grants-Program/blob/master/grants/grant-badge-guidelines.md). The badge can also never be used as a general endorsement for a team. Violations to this policy can be reported [here](mailto:grants@web3.foundation).
```

File: grants/accepted_grant_applications.md (hunk 1 above)
Comment (Lederstrumpf) [reply grants/accepted_grant_applications.md:1]: Agree with Alex

**Review by Lederstrumpf — APPROVED** [review]
LGTM, including suggested rephrasings from @alxs

File: grants/accepted_grant_applications.md (hunk 1 above)
Comment (Lederstrumpf) [reply grants/accepted_grant_applications.md:1]: I thought about phrasing this to make it explicit that this is a living document, but given the document's purpose, it should be obvious to any reader.

Comment (alxs) [reply grants/accepted_grant_applications.md:1]: Those were my thoughts also

Comment (Noc2) [reply grants/accepted_grant_applications.md]: Sorry, I removed it. The only purpose of it was that teams already reached out to me yesterday and complained about the status of this document.

Comment (Noc2) [reply grants/accepted_grant_applications.md]: So I quickly added WIP

**Review by semuelle — APPROVED** [review]

**Pull request merged**